docs/changelog.rst

```diff
@@ -1,6 +1,15 @@
 Changelog
 ---------
 
+0.11.7
+^^^^^^
+Wednesday April 19 2019
+
+* Update facade methods for Juju 2.6
```

Comment:
2.6.3? 2.6.4?

I think we should be explicit here. Not sure if we should go so far as to a sha it was built against or not though...